base/Enums.jl

```diff
@@ -29,7 +29,7 @@ Base.print(io::IO, x::Enum) = print(io, Symbol(x))
 
 function Base.show(io::IO, x::Enum)
     sym = Symbol(x)
-    if !get(io, :compact, false)
+    if !(get(io, :compact, false)::Bool)
```

Comment:
Fortunately `::` already binds tighter than `!`, but I can see how that might not be obvious to code readers.

Reply:
i can change it, but I felt that obviousness was worth a few extra characters.